lib/export_dsi_users_to_big_query.rb

```diff
@@ -23,7 +23,8 @@ def present_for_big_query(batch)
         given_name: user['givenName'],
         family_name: user['familyName'],
         email: user['email'],
-        school_urn: user.dig('organisation', 'URN')
+        school_urn: user.dig('organisation', 'URN'),
+        organisation_uid: user.dig('organisation', 'UID')
```

Comment:
Could we make 'urn' and 'URN' consistent between the two tasks (and the json)?

Reply:
When we receive them it's capitalised for users and lowercase for approvers. I'm not too sure at which point I could downcase it?

Reply:
Ah, I see. I assumed DSI would be consistent. Silly me.